cns/NetworkContainerContract.go

```diff
@@ -406,6 +422,10 @@ type PodIpInfo struct {
 	PodIPConfig                     IPSubnet
 	NetworkContainerPrimaryIPConfig IPConfiguration
 	HostPrimaryIPInfo               HostIPInfo
+	NICType                         NICType // default, secondary
```

Comment:
lets remove comment as //default, secondary. We are changing names to be {Infra, Secondary, Management and IB}